elastic/package-registry
# Expose policy_templates_behavior and deployment modes for policy templates in search and package endpoints

Closes https://github.com/elastic/package-registry/issues/1238
Closes https://github.com/elastic/package-registry/issues/1243

Expose `policy_templates_behavior` and the `deployment_modes` for the policy templates (if any) in both search and package endpoints.

## How to test

1. Compile and run Elastic Package Registry with the test packages (using `config.yml` from the repo):
   ```shell
   eval "$(gvm $(cat .go-version))"
   go run . -config config.yml
   ```
2. In another terminal, run these queries and ensure those new fields are present:
   ```shell
   curl -s "http://127.0.0.1:8080/search?package=deployment_modes&prerelease=true"
   curl -s "http://127.0.0.1:8080/package/deployment_modes/0.0.1/"
   ```
